src/components/film-filter/film-filter.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import {
+  Row, Col, Dropdown, DropdownToggle,
+} from 'reactstrap';
+import PropTypes from 'prop-types';
+
+import { useFilters } from '../../hooks/use-filters';
+import { DropDownMenu } from '../drop-down-menu/drop-down-menu';
+import { IconArrow } from '../icons/icon-arrow';
+
+export const FilmFilter = ({
+  items, defaultValue, filterName,
+}) => {
+  const { dispatchAddNewFilter, dispatchRemoveFilter, filters } = useFilters();
```

Comment:
Why should I use hook in `FilterFilm` instead of `FilterFilms`. 
Hook returns array of filters and `FilterFilm` renders single filter. Should I transfer hook to `FilterFilms` component and pass single filter to `FilterFilm`?
So, `FilterFilm` has `items`. Should I pass `selectItem` (like `onItemClick`) to `FilterFilm` or  create it inside `FilterFilm` using `dispatchRemoveFilter` and `dispatchAddNewFilter ` functions from hook?